tests/test_surface/test_regular_surface.py

```diff
@@ -34,6 +34,8 @@
 TESTSET6A = TPATH / "surfaces/etc/seabed_p.pmd"
 TESTSET6B = TPATH / "surfaces/etc/seabed_p.gri"
 TESTSET6C = TPATH / "surfaces/etc/seabed_p_v2.pmd"
+TESTSET7A = TPATH / "surfaces/etc/topvolantis_genhash.gri"
+TESTSET7B = TPATH / "surfaces/etc/toptherys_genhash.gri"
```

Comment:
If you want to include additional test data, that is done in the repository `xtgeo-testdata`

Reply:
Ah sorry, saw the PR now!

Reply:
Yep! I hope the naming/location is fine enough. It may have been a better fit in `/reek` but the single test use-case of the files seemed fairly `/etc`. I'll just ask next time.

Reply:
naming and location is ok, topvolantis is not the old reek anyway